backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/JniScanProgress.kt

```diff
@@ -17,8 +17,10 @@ class JniScanProgress(
         require(numerator >= 0L) {
             "Numerator $numerator is outside of allowed range [0, Long.MAX_VALUE]"
         }
-        require(denominator >= 1L) {
```

Comment:
This should be okay now that you've fixed it in the Rust layer.

Reply:
Yep, I then reverted this change back in [1b6039c](https://github.com/zcash/zcash-android-wallet-sdk/pull/1116/commits/1b6039c36800beea17dbe156e85e1e4f45c1212f).